src/actions/NewSensorActions.js

```diff
@@ -0,0 +1,7 @@
+export const NEW_SENSOR_ACTIONS = {
+  SELECT: 'SensorActions/select',
+};
+
+const select = macAddress => ({ type: NEW_SENSOR_ACTIONS.SELECT, payload: { macAddress } });
+
+export const NewSensorActions = { select };
```

Comment:
will `NewSensorActions` will become `SensorActions` at a later stage?
the const is `NEW_SENSOR_ACTIONS` though the text string is `SensorActions` which made me think that this file is for sensor actions. not for new sensors.

Reply:
I thought this particular case would stay as `new` as it is the 'new' sensor flow/feature. Happier? Sadder?

Reply:
sill vaguely confused.. but I think all good. you know what you're doing.
this is for working with new sensors, so the 'New' is appropriate and will stay. is that it?

Reply:
Yeah exactly (except for the part of me knowing what I am doing) my thinking - I figure it will be clearer when there aren't the .. "other new prefixes" confusing things...